test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -36,6 +36,7 @@
 # trademark licensing policies.
 
 from tests.functional import *
+import os
```

Comment:
The os module is already loaded. No need to do so here.

Reply:
Removed. Thanks.